reproman/distributions/tests/test_debian.py

```diff
@@ -28,6 +26,8 @@
 
 
 @mark.skipif_no_apt_cache
+@pytest.mark.skipif("cmd:ed" not in external_versions,
```

Comment:
I can indeed see minimized environments having no `ed`. I am just curious -- was it some real system without `ed` installed?

Reply:
I like to think my laptop is a real system :]

From the commit message of e2ec169a5:

> This change assumed that Ed is installed, which isn't the case for my local system.

Reply:
Oh, and you don't even have to look at the commit messages :]  From the PR description:

> test_dpkg_manager_identify_packages was failing for me locally because I didn't have Ed installed on my system.